BLSQ/iaso
# IA-3387: Implement OUCRC list screen

Implement OrgUnit Change Request Configuration list screen

Related JIRA tickets : IA-3387

## Self proofreading checklist


## Changes

This adds a new entry in the menu and a new list for the OrgUnit Change Request Configurations

## How to test

You can either wait for the backend to be ready of use the fixtures.
You can also see it in action in the Cypress tests.
